bananagrams-ui/src/setupTests.ts

```diff
@@ -0,0 +1,6 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
+// allows you to do things like:
+// expect(element).toHaveTextContent(/react/i)
+// learn more: https://github.com/testing-library/jest-dom
```

Comment:
Everything jest related is for unit testing - I'm happy to try to learn front end unit testing but we sniped this in the other project I'm working on (and all the other unit testing things)